builtins/amp-video.js

```diff
@@ -98,6 +121,80 @@ export function installVideo(win) {
     isVideoSupported_() {
       return !!this.video_.play;
     }
+
+    /**
+     * VideoInterface Implementation
+     * {@see ../src/video-interface.VideoInterface}
+     */
+
+    /**
+     * @return {boolean}
+     */
+    supportsPlatform() {
+      return this.isVideoSupported_();
+    }
+
+    /**
+     * @return {boolean}
+     */
+    hasAutoplay() {
+      return this.element.hasAttribute('autoplay');
+    }
+
+    /**
+     * @return {!Promise}
+     */
+    play(unusedIsAutoplay) {
+      this.video_.play();
+      return listenOncePromise(this.video_, 'play');
+    }
+
+    /**
+     * @return {!Promise}
+     */
+    pause() {
+      this.video_.pause();
+      return listenOncePromise(this.video_, 'pause');
```

Comment:
Ditto, if it's not currently playing. E.g. because it finished or already paused?